scripts/jlab-env.csh.in

```diff
@@ -1 +1,2 @@
 module load gcc/@CMAKE_C_COMPILER_VERSION@
+module load python/3.9.5
```

Comment:
Shouldn't this line put /group/hps/hps_soft/python3.9.5/site-packages in the sys.path for you? Not clear why you need this and also to append in batch.py.

Reply:
No, the module command will not be aware of custom stuff installed in the hps account without modulefiles and without registering those modulefiles.

Reply:
This line I understand, it is really the append that isn't needed if you really install the package for python correctly.